src/conftest.py

```diff
@@ -128,23 +130,27 @@ def clean_crate():
 
 @pytest.fixture()
 def crate_translator(clean_crate):
-    from src.translators.crate import CrateTranslator
+    from src.translators.crate import CrateTranslator, CrateConnectionData
+    crate_host = os.environ.get('CRATE_HOST', 'crate')
+    crate_port = 4200
+    crate_db_username = os.environ.get('CRATE_DB_USER', 'crate')
+    crate_db_password = os.environ.get('CRATE_DB_PASS', None)
 
     class Translator(CrateTranslator):
 
         def insert(self, entities,
-                   fiware_service=None, fiware_servicepath=None):
+                    fiware_service=None, fiware_servicepath=None):
```

Comment:
Extra spaces here?

Reply:
could be... next autopep8 will fix, unfortunately, on external pull requests we can't enforce autopep8 since we don't own the repo...